geometry/clipping.cpp

```diff
@@ -71,29 +75,26 @@ bool IntersectEdge(m2::RectD const & rect, std::vector<m2::PointD> const & corne
   }
   return false;
 }
+} // namespace
 
 void ClipTriangleByRect(m2::RectD const & rect, m2::PointD const & p1, m2::PointD const & p2,
                         m2::PointD const & p3, ClipTriangleByRectResultIt const & resultIterator)
 {
-  if (resultIterator == nullptr)
-    return;
-
   if (rect.IsPointInside(p1) && rect.IsPointInside(p2) && rect.IsPointInside(p3))
   {
     resultIterator(p1, p2, p3);
     return;
   }
 
-  static double constexpr kEps = 1e-8;
+  double constexpr kEps = 1e-8;
```

Comment:
`static constexpr` is better than `constexpr`: https://stackoverflow.com/a/13867690

Reply:
I saw this article and there is no any reason to use static here, however from your link:
```
However, there is one case where you wouldn't want to use static constexpr. Unless a constexpr declared object is either [ODR-used](https://en.cppreference.com/w/cpp/language/definition#ODR-use) or declared static, the compiler is free to not include it at all. That's pretty useful, because it allows the use of compile-time temporary constexpr arrays without polluting the compiled program with unnecessary bytes. In that case, you would clearly not want to use static, since static is likely to force the object to exist at runtime.
```

I suspect that static or not static for constexpr integrals will be the same - the compiler should inline this constant.